extensions/podman/packages/extension/src/utils/podman-cli.ts

```diff
@@ -31,19 +31,49 @@ export async function findPodmanInstallations(): Promise<string[]> {
       // Windows: Use 'where podman' command
       result = await extensionApi.process.exec('where', ['podman']);
     } else {
-      // Unix/macOS: Use 'type -a podman' command
-      result = await extensionApi.process.exec('sh', ['-c', 'type -a podman']);
+      // Unix/macOS: Try which first, then whereis as fallback
+      let whichError: Error | undefined;
+      try {
+        // Try 'which -a podman' first
+        result = await extensionApi.process.exec('which', ['-a', 'podman']);
+      } catch (error) {
+        whichError = error as Error;
+        try {
+          // Fallback to 'whereis podman' if which is not available
+          result = await extensionApi.process.exec('whereis', ['podman']);
```

Comment:
Why do we need this ? There are plenty of other pieces of code in Podman Desktop that uses where on Windows or which on Linux/macOS

Reply:
Good to know, I wanted to make it as safe as possible. I can absolutely reduce it to just `where` use.